app/src/componentLibrary/autocompletes/multipleAutocomplete/multipleAutocomplete.jsx

```diff
@@ -203,7 +204,10 @@ export const MultipleAutocomplete = ({
                 updatePosition = scheduleUpdate;
                 const filteredOptions = options.filter(
                   (item) =>
-                    value.indexOf(item) < 0 && item.toLowerCase() !== inputValue.toLowerCase(),
+                    value.indexOf(item) < 0 &&
```

Comment:
Could we give this expression a more descriptional name?
It is absolutely hard to figure out what we are doing here.
And, after that, maybe we could find an easier way to handle this scenario.